zircon.core/common/src/main/kotlin/org/hexworks/zircon/api/builder/component/HorizontalNumberInputBuilder.kt

```diff
@@ -15,29 +15,45 @@ import kotlin.math.max
 data class HorizontalNumberInputBuilder(
         val width: Int,
         private var initialValue: Int = 0,
+        private var minValue: Int = 0,
```

Comment:
Is it possible to have negative numbers here?

Reply:
I totally did not check if that could work, but I don't think it would cause problems

Reply:
So the answer is no, looks like it introduces bugs in the rendering of the slider. Must be relatively easy to fix I guess. For now there are checks in the builder to not let users do it.